pandas/tests/plotting/test_frame.py

```diff
@@ -204,6 +204,23 @@ def test_color_and_style_arguments(self):
         with pytest.raises(ValueError):
             df.plot(color=["red", "black"], style=["k-", "r--"])
 
+    def test_color_and_marker(self):
+        # GH21003 - code sample from 2018-05-10 is equivalent to this test
+        df = DataFrame(np.random.random((7, 4)))
+        # combining a color string and a marker letter should be allowed
+        df.plot(color="green", style="d")  # d for diamond
```

Comment:
could we also check `color` and `style` here?
something like:
```python
ax = df.plot(color="green", style="d")
assert ax.lines.get_color() == 'green'
assert ax.lines.get_marker() == 'd'
```

Reply:
see my reply below.